test/index.js

```diff
@@ -32,12 +32,22 @@ const changeCwd = function (cwd) {
   return process.chdir.bind(process, originalCwd)
 }
 
+// Move .next from sample project to current directory
+const moveNextDist = function () {
+  // Use copySync because cpx doesn't copy hidden files
+  fs.copySync(`${FIXTURES_DIR}/sample/.next`, `${process.cwd()}/.next`)
+}
+
 // Copy fixture files to the current directory
 const useFixture = async function (fixtureName) {
   const fixtureDir = `${FIXTURES_DIR}/${fixtureName}`
   await pCopy(`${fixtureDir}/**`, process.cwd())
 }
 
+beforeAll(async () => {
+  await npmRun('build', `${FIXTURES_DIR}/sample`)
+})
```

Comment:
Should we merge the fixture's `package.json` with the top-level `package.json`?
However, we would need to update dependencies on several `package.json` and run `npm install` on several directories.

Reply:
next-on-netlify generally follows this model of having (full) sample projects within tests/fixtures, it's all needed to run the build from within the tests i think. i can move sample up one dir into `test/` but i think it better belongs there than top level, which is where it used to be. wherever it is, it's the convenient sample project to run the plugin against as a local plugin, so it needs its own package.json

Reply:
Having full test fixtures sound good. I am just wondering if we could keep the set of files to the minimum required for the test to work. For example, `.gitignore` might not be needed.